scripts/openzfs-merge.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash
+#
+# This script tries to merge OpenZFS commits to ZoL.
+#
+# Instruction:
+# 
+# Repository setup must be similar with openzfs-tracking.sh
+# requirements.
+#
+# Repository setup for valid compilation check:
+#	sh autogen.sh
+#	./configure --enable-debug
```

Comment:
It will we safer, but much more slower.

I didn't have any issues with it yet.